app/assets/stylesheets/custom.scss

```
@@ -109,4 +115,53 @@
 
 .tm-revcon:focus {
   outline: none;
+  @mixin box_sizing {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+  }
+}
```

```suggestion
  outline: none;
  @mixin box_sizing {
    -moz-box-sizing: border-box;
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
  }
}
```
  @include box_sizing;